ecommers/lib/core/app_services/dialog/confirm_phone_registration.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:ecommers/core/app_services/dialog/dialog_base.dart';
+import 'package:ecommers/core/services/dependency_service.dart';
+import 'package:ecommers/ui/decorations/index.dart';
+import 'package:flutter/material.dart';
+
+class ConfirmPhoneRegistration extends DialogBase<String> {
```

Comment:
Maybe it would be better to put dialogBase classes in presentation layer?

Reply:
I will move it in next PR's